invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/administration/components/SetQuotaAction.js

```diff
@@ -0,0 +1,92 @@
+/*
+ * // This file is part of Invenio-App-Rdm
+ * // Copyright (C) 2023 CERN.
+ * //
+ * // Invenio-App-Rdm is free software; you can redistribute it and/or modify it
+ * // under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { Button, Modal, Icon } from "semantic-ui-react";
+import { ActionModal } from "@js/invenio_administration";
+import _isEmpty from "lodash/isEmpty";
+import { SetQuotaForm } from "./SetQuotaForm";
+
+export class SetQuotaAction extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      modalOpen: false,
+      modalHeader: undefined,
+      modalBody: undefined,
+    };
+  }
+
+  onModalTriggerClick = (e, { payloadSchema, dataName, dataActionKey }) => {
+    const { resource, apiUrl } = this.props;
+
+    this.setState({
+      modalOpen: true,
+      modalHeader: "Set quota",
```

Comment:
Mark for translation? There are multiple other strings not marked for translation in the PR.